cli/src/types.rs

```diff
@@ -126,6 +129,289 @@ pub struct RevokeTokenRequest<'a> {
     pub name: &'a str,
 }
 
+/// Response body for `/data/packages/submit`.
```

Comment:
I touched up some of these doc comments and removed ones that seemed redundant... lmk what you think.